package.json

```diff
@@ -51,7 +52,6 @@
     "babel-preset-react": "^6.24.1",
     "enzyme": "^3.7.0",
     "enzyme-adapter-react-16": "^1.7.0",
-    "jest": "^23.6.0",
```

Comment:
Why was Jest shifted out of the dev dependencies?

Reply:
My only guess would be when I tried reinstalling Jest to get tests working.

Reply:
I have moved it back to being a dev dependency now.